docs/lib/geo/fragments/js/search.md

```diff
@@ -22,30 +22,22 @@ First, create a map onto which you want to add the location search UI component.
 
 The location search UI component (`maplibre-gl-geocoder`) requires a "geocoding API" to facilitate location-based search. `AmplifyGeocoderAPI` is geocoding API that is integrated with Amplify Geo. To define your own "geocoding API", see [AmplifyGeocoderAPI](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/src/AmplifyMapLibreGeocoder.ts).
```

Comment:
I still see this, do we need it?

Reply:
Added a line about how createAmplifyGeocoder creates a geocoder with pre defined settings

Reply:
I would probably be more on the side of just removing this, unless we are also adding a way for them to add a custom geocoder as well, though I didn't see that.

Reply:
Whoops actually I realized I misread your last comment, fixed